src/cairoUtilFuncGen/calldata/calldataToMemory.ts

```diff
@@ -96,44 +96,40 @@ export class CallDataToMemoryGen extends StringIndexedFuncGen {
     if (isReferenceType(elementT)) {
       const recursiveFunc = this.getOrCreateFuncDef(elementT);
       copyCode = endent`
-        let cdElem = calldata[0];
-        let (mElem) = ${recursiveFunc.name}(cdElem);
-        dict_write{dict_ptr=warp_memory}(mem_start, mElem);
+        let calldata_elem = calldata[index];
+        let memory_elem = ${recursiveFunc.name}(calldata_elem);
+        warp_memory.unsafe_write(calldata_elem, memory_elem)
       `;
       auxFunc = recursiveFunc;
     } else if (memoryElementWidth === 2) {
       copyCode = endent`
-        dict_write{dict_ptr=warp_memory}(mem_start, calldata[0].low);
-        dict_write{dict_ptr=warp_memory}(mem_start+1, calldata[0].high);
+        warp_memory.unsafe_write(mem_ptr, calldata[index].low)
+        warp_memory.unsafe_write(mem_ptr + 1, calldata[index].high)
```

Comment:
Wouldn't `warp_memory.store(mem_ptr, calldata[index])` be better here?

Reply:
Better in a readability way, I don't think it would be better in any other way.